src/psevents.c

```diff
@@ -167,79 +167,93 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSEVENTS_CTRL *C) {	/* Deall
-	GMT_Message (API, GMT_TIME_NONE, "\t   or offset, rise,, and fade intervals for text (-Et):\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +o<dt> offsets event start and end times by <dt> [no offset].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     Use +O<dt> to only offset event start time and leave end time alone.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +r<dt> sets the rise-time before the event start time [no rise time].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +p<dt> sets the length of the plateau after event happens [no plateau].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +d<dt> sets the decay-time after the plateau [no decay].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +f<dt> sets the fade-time after the event ends [no fade time].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +l<dt> sets alternative label duration [same as symbol].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t-F Specify values for text attributes that apply to all text records:\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +a<angle> specifies the baseline angle for all text [0].\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t     Use +A to force text-baselines in the -90/+90 range.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   +f<fontinfo> sets the size, font, and optionally the text color [%s].\n",
+	GMT_Usage (API, -2, "Record format: lon lat [z] [size] time [length|time2].");
+	GMT_Usage (API, 1, "\n-Ar[<dpu>[c|i]]|s");
+	GMT_Usage (API, -2, "Select plotting of lines or polygons when no -S is given.  Choose input mode:");
+	GMT_Usage (API, 3, "r: Read records for lines with time in column 3. "
+		"Append <dpu> to convert your line records into dense point records that can be plotted as circles. "
+		"The resampled line will be written to stdout (requires options -R -J, optionally -C). "
+		"The <dpu> must be the same as the intended <dpu> for the movie frames. "
+		"Append i if dpi and c if dpc [Default will consult GMT_LENGTH_UNIT setting, currently %s].", API->GMT->session.unit_name[API->GMT->current.setting.proj_length_unit]);
+	GMT_Usage (API, 3, "s: Read whole segments (lines or polygons) with no time column. n"
+		"Time is set via segment header -T<start>, -T<start>,<end>, or -T<start>,<duration (see -L).");
+	GMT_Usage (API, 1, "\n-C<cpt>");
+	GMT_Usage (API, -2, "Give <cpt> and obtain symbol color via z-value in 3rd data column.");
+	GMT_Usage (API, 1, "\n-D[j|J]<dx>[/<dy>][+v[<pen>]");
+	GMT_Usage (API, -2, "Add <add_x>,<add_y> to the event text origin AFTER projecting with -J [0/0]. "
+		"Use -Dj to move text origin away from point (direction determined by text's justification). "
+		"Upper case -DJ will shorten diagonal shifts at corners by sqrt(2). "
+		"Append +v[<pen>] to draw line from text to original point.  If <add_y> is not given it equals <add_x>.");
+	GMT_Usage (API, 1, "\n-E[s|t][+o|O<dt>][+r<dt>][+p<dt>][+d<dt>][+f<dt>][+l<dt>]");
```

Comment:
```suggestion
	GMT_Usage (API, 1, "\n-Es|t[+o|O<dt>][+r<dt>][+p<dt>][+d<dt>][+f<dt>][+l<dt>]");
```
EIther this or rst docs need updating

Reply:
Fixed the rst.